.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots or example input/output images to help explain your problem.
+
+**Additional context**
+Add any other context about the problem here (e.g. OS).
```

Comment:
I think a lot of bugs are related to the environment (compiler version, compile flags used, CPU where you run it) specially related to the SIMD code. We should explicitly ask for those.

Reply:
RIght. PTAL if this is better.